packages/one-app-bundler/__tests__/webpack/loaders/common.spec.js

```diff
@@ -123,6 +124,81 @@ describe('Common webpack loaders', () => {
       getConfigOptions.mockReturnValueOnce({ purgecss: { disabled: true } });
       expect(purgeCssLoader()).toEqual([]);
     });
+    it('should include purgecss 3 configuration options', () => {
+      const purgecss = {
+        paths: ['foo', 'bar'],
+        extractors: [{
+          extractor: 'purgeJs',
+          extensions: ['js'],
+        }],
+        fontFace: true,
+        keyframes: true,
+        variables: true,
+        safelist: {
+          standard: ['random'],
+          deep: ['randomdeep'],
+          greedy: ['randomgreedy'],
+          keyframes: true,
+          variables: true,
+        },
+        blocklist: ['blockClass'],
+      };
+      getConfigOptions.mockReturnValueOnce({ purgecss });
+      expect(purgeCssLoader()).toMatchSnapshot();
```

Comment:
Can we be more intentional with these tests? Let's test specific fields rather than relying on a snapshot. It makes the test more understandable to the reader, and gives me a bit more confidence